### ISSUE TITLE
batch size question

### ISSUE THREAD

**Issue by free-soellingeraj** [timeseriesAI/tsai#40]
Hi, I am looking at the `TSDataLoaders` and wondering what is the role of the validation batch size?  I am used to setting up the training batch size, which defines the number of updates (1 per row) of each epoch.  But in the past, for example OOB error, the validation set is just either all computed, or sampled.  Here's what I am doing.  Or possibly, I am mis-interpreting the 2 elements fed into `bs=..` as being training and validation.  
```python
dls = TSDataLoaders.from_dsets(
    ds.train, ds.valid, bs=[train_bs, valid_bs], shuffle_train=False
)
```
thanks for your work.

**Comment by oguiza** [comment]
When creating a TSDataLoaders you can pass a bs as a single integer, or a list of integers of the same length as the splits (train, valid, maybe test).
The reasons: 
1. Backpropagation of the training data requires gradients and uses more memory. But backprop is not used with the valid set. That’s why you can make it larger without running the risk of running out of cuda memory, and the process will be faster. 
2. bs in the training set has an impact on how the model is trained. Higher bs may require a higher lr for example. But valid bs doesn’t have that impact, and you want to run it as fast as you can as long as data fit in cuda memory.
